Sources/Overlays/_Testing_CoreGraphics/Attachments/AttachableAsCGImage.swift

```diff
@@ -39,7 +39,7 @@ private import ImageIO
 ///   @Available(Swift, introduced: 6.3)
 /// }
 @available(_uttypesAPI, *)
-public protocol AttachableAsCGImage: SendableMetatype {
+public protocol AttachableAsCGImage: _AttachableAsImage, SendableMetatype {
```

Comment:
Since `SendableMetatype` is now stated in the parent protocol, does it still make sense to reiterate here?